backend/lib/endpoints/schema/posts.js

```diff
@@ -13,6 +13,8 @@ const getPostsSchema = {
     .prop("actorId", S.string())
     .prop("authorId", S.string())
     .prop("filter", S.string()) // URI encoded JSON; TODO: figure out way to custom validation
+    .prop("sort", S.string()) // Sorting posts by likes, updatedAt, views and shares
```

Comment:
it's better to add strict validation for the inputs, since it goes directly to the DB querying, could refer to the [Enum options like this one](https://github.com/FightPandemics/FightPandemics/blob/50edf4a5b84f16dbbffb864e85772715275023b3/backend/lib/endpoints/schema/posts.js#L9)